docs/docs/latest/tutorials/getting-started.md

```diff
@@ -25,15 +25,15 @@ Windows. For other options, such as Docker, read the [detailed instructions] (/d
 1. Download the `edgectl` file [here](https://metriton.datawire.io/downloads/darwin/edgectl) or download it with a curl command:
 
     ```shell
-    sudo curl -fL https://metriton.datawire.io/downloads/darwin/edgectl -o /usr/local/bin/edgectl && sudo chmod a+X /usr/local/bin/edgectl
+    sudo curl -fL https://metriton.datawire.io/downloads/darwin/edgectl -o /usr/local/bin/edgectl && sudo chmod a+x /usr/local/bin/edgectl
     ```
 
     If you decide to download the file, you may encounter a security block. To change this:
     * Go to **System Preferences > Security & Privacy > General**.
     * Click the **Open Anyway** button.
     * On the new dialog, click the **Open** button.
 
-2. Run the installer with `./edgectl install`
```

Comment:
Why did you remove the ./ ? Many people's PATH does not include . and thus one has to use ./edgectl rather than just edgectl in order to execute it.

Reply:
this is some of the code that I somehow inherited--I didn't make those changes, so wanted to figure out how to separate out my changes from the others.

Reply:
this is no longer the case now that the PR is correctly branched off of release/v1.3.2

Reply:
@bnfb the commands to download and adjust permissions for the binary actually place it in /usr/local/bin so in fact attempting to run the command with a prefix of "./" will fail.